sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/ItCoverageResultImporter.java

```diff
@@ -19,14 +19,14 @@
  */
 package org.sonar.plugins.php.phpunit;
 
-import org.sonar.api.batch.fs.FileSystem;
 import org.sonar.api.batch.sensor.coverage.CoverageType;
 import org.sonar.api.measures.CoreMetrics;
+import org.sonar.plugins.php.PhpPlugin;
 
-public class PhpUnitItCoverageResultImporter extends PhpUnitCoverageResultImporter {
+public class ItCoverageResultImporter extends CoverageResultImporter {
 
-  public PhpUnitItCoverageResultImporter(FileSystem fileSystem) {
-    super(fileSystem);
+  public ItCoverageResultImporter() {
+    super(PhpPlugin.PHPUNIT_IT_COVERAGE_REPORT_PATH_KEY, "integration test coverage");
     linesToCoverMetric = CoreMetrics.IT_LINES_TO_COVER;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "IT_LINES_TO_COVER"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)